src/main/scala/io/iohk/ethereum/extvm/VMServer.scala

```diff
@@ -180,6 +180,7 @@ class VMServer(messageHandler: MessageHandler)
       eip150BlockNumber = conf.eip150BlockNumber,
       eip160BlockNumber = conf.eip160BlockNumber,
       eip161BlockNumber = conf.eip161BlockNumber,
+      byzantiumBlockNumber = conf.byzantiumBlockNumber,
```

Comment:
unfortunatly byzantium number is not defined in protobuf yet, i have already asked IELE guys do they have some process to add it there.